src/source/Crypto/Dtls.h

```diff
@@ -71,31 +75,66 @@ typedef struct {
   SRTP_PROFILE srtpProfile;
 } DtlsKeyingMaterial, *PDtlsKeyingMaterial;
 
+#ifdef KVS_USE_OPENSSL
 typedef struct {
     BOOL created;
     X509 *pCert;
     EVP_PKEY *pKey;
 } DtlsSessionCertificateInfo, *PDtlsSessionCertificateInfo;
 
+#elif KVS_USE_MBEDTLS
```

Comment:
Minor: This should be else. If you are using elif then perhaps you should have the else clause with an error that the TLS is not defined

Reply:
Ah, forgot that C has #error directive. I'll do that. Thanks.

Reply:
Done